CosmosToRedis/CosmosToRediscs.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.Azure.Cosmos;
```

Comment:
File rename CosmosToRediscs.cs -> CosmosToRedis.cs

Reply:
I believe this file is being re-written with the read-through so won't review further

Reply:
Is this file actually being used?

Reply:
No, I use the CosmosToRedis, so I will remove the other one.